get_session_request.php

```diff
@@ -0,0 +1,64 @@
+<?php
+require_once 'vendor/autoload.php';
```

Comment:
According to the PHP documentation, [`require_once` searches for the specified file in the calling script's directory and the current working directory](https://www.php.net/manual/en/function.include.php). Since `build_artifacts.sh` doesn't currently copy the `vendor` folder over to the build dir, I suppose that means you started `php` from outside the build directory, when running locally? That means that this is going to break when deployed.

Adding this to `build_artifacts.sh` near the end of the for loop solves this:
```sh
cp -r "$DIR/vendor" "$DIR/build/$lang/vendor"
```

Reply:
done. also added `data` which I forgot about -.-